misc/install-deps.sh

```diff
@@ -1,5 +1,41 @@
 #!/bin/sh
 
+install_packages() {
+    case $1 in
+        "ubuntu" | "debian" | "raspbian" | "kali" | "linuxmint")
```

Comment:
I think we can remove "raspbian", "kali", "linuxmint" as they are variants of former two and have ID_LIKE field.

Reply:
Okay, I just amended the commit :)